### ISSUE TITLE
Sale fulfillment - allow selecting "no group"

### ISSUE THREAD

**Issue by altemeier** [agex-inc/support.requests#356]
# Report

### What version of the app were you using?
iOS 189, Android 21800

### In what screen/page did you find the bug?
Sale Fulfillment - Step 1 - Select Group

### What actually happened?
On Android, I was not given the option to change sale groups with only one group in my Herd.  iOS displayed a groups dropdown with the only Herd group "preselected", but when I tapped the dropdown the only "option" was "no groups", and I could not select it.

Consider a dropdown with the current group and "no group" as the two selectable options.

### How severe is this issue?
LOW: would be nice to fix when we can

### Timestamp
8/21/2020 10:18:07

### What device were you using?
Multiple

### What part of the system needs work?
Herd

### What were you generally trying to do?
"Unselect" the only group in my Herd

### Is this bug also in the DEMO and/or PRODUCTION environments?
Yes, the bug also occurs in DEMO and/or PRODUCTION

### Email Address
salt@agex.io

### Is this a bug or a new feature request?
Bug (something unexpected is happening)

### What environment were you working in?
TEST

### Were you using a "native" or "web" app?
Native (downloaded from a store)

### What was the email (or phone number) for the account you were using?
8.19.operator@testios.com

**Comment by eharo2** [comment]
@altemeier @joshpavlovich    If the sale has a group attached and the user selects "No group" at fulfill time, beside removing the group_id from the sale (and dissociating the  sale in the group), should the animal_profile be also removed?    If we do, it would be similar to publishing a sale without a group and without a definition of headcount and weight, that would complicate any price analysis in the sale.....

Mi recommendation is to leave the the animal profile as is, given that it is key in the sale contract....     So if the group is deselected, the animals view at fulfill would be "3 head, steer".     Do you agree?

**Comment by joshpavlovich** [comment]
@eharo2 I agree that the animal profile should not be cleared if "None" (a.k.a "No group") is selected.

**Comment by eharo2** [comment]
@altemeier @joshpavlovich We are in agreement... I will implement it this way and if there is any change we can discuss... e
